src/lib/falcor/FalcorRouter.js

```diff
@@ -914,6 +915,106 @@ export default class FalcorRouter extends BaseRouter.createClass([
     },
   },
   {
+    route: "search['teams'][{keys:queries}][{integers:indices}]",
+    get: (pathSet) => {
+      return new Promise((resolve) => {
+        let minIndex = pathSet.indices[0];
+        let maxIndex = pathSet.indices[0];
+        pathSet.indices.forEach((index) => {
+          if (index < minIndex) {
+            minIndex = index;
+          }
+          if (index > maxIndex) {
+            maxIndex = index;
+          }
+        });
+        db.searchTeamsQuery(pathSet.queries, minIndex, maxIndex).then((data) => {
+          // Map all the indices down to fit the indices returned by the db call
+          pathSet.indices = pathSet.indices.map((index) => {return index-minIndex});
+          const results = [];
+          _.forEach(data, (queryResults, query) => {
+            pathSet.indices.forEach((index) => {
+              if (index < queryResults.length) {
+                results.push({
+                  path: ['search', 'teams', query, index],
+                  value: $ref(['teamsBySlug', queryResults[index]]),
+                });
+              }
+            });
+          });
+          resolve(results);
+        });
+      });
+    },
+  },
+  {
+    route: "teamsBySlug[{keys:slugs}]['id', 'name', 'description']",
```

Comment:
@emilgoldsmith when mapping each field in
```
{
                path: ["teamsBySlug", team.slug, field],
                value: team[field],
              }
```
this adds the slug everywhere. Are you sure you want me to add the slug anyway? Also, if I do, should I exclude it in this mapping to avoid redundancy?